src/javascripts/ng-admin/Crud/misc/maHelpText.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Help text for a form field.
+ *
+ * @example <ma-help-text field="field"></ma-help-text>
+ *
+ * @param field.helpText()  If a string, it will be displayed as text to the user.
+ *                          If an `angular.element`, it will be compiled and displayed.
+ *                          You can use "entry.values[x]" to check the current value
+ *                          of property "x" (e.g. if you want to give different hints)
+ */
+export default function maHelpText($compile) {
+    return {
+        scope: true,
+        restrict: 'E',
+        link: function(scope, element) {
+            var field = scope.field;
+
+            var helpText = field && field.helpText();
+            if (helpText && typeof helpText === "object" && helpText.contents) {
```

Comment:
It should be the default case. First, I would test if helpText is a string, then a function, then consider it is a custom directive.